### PR TITLE
Feature/feed

### PR DESCRIPTION
**Pull request by GarageInc**
**Summary**

Add feed page. Atm only with activities

**Screenshots**
image

image
image
image
image
image

### CODE REVIEW

File: src/pages/feed/GeneralFeed/GeneralFeed.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import SonarFeedActivityPage from '../../SonarFeed/SonarFeedActivityPage'
+import styles from './GeneralFeed.module.scss'
+import HelpTooltip from '../../../components/WatchlistOverview/WatchlistAnomalies/HelpTooltip'
```
Comment (haritonasty) [inline src/pages/feed/GeneralFeed/GeneralFeed.js]: rearrange imports

File: src/pages/feed/GeneralFeed/GeneralFeed.module.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
+@import '~@santiment-network/ui/mixins';
+
+.title {
+  @include text('body-1');
+
+  display: flex;
+  color: $rhino;
```
Comment (haritonasty) [inline src/pages/feed/GeneralFeed/GeneralFeed.module.scss]: `color: var(--rhino);` for night mode, please

File: src/pages/feed/GeneralFeed/GeneralFeed.module.scss [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+.activityCustom {
+  margin: 0 0 16px 0 !important;
+}
+
+.activityMarkdown {
+  p {
+    // GarageInc: becuase markdown from server
+    margin-bottom: 0 !important;
+  }
```
Comment (haritonasty) [inline src/pages/feed/GeneralFeed/GeneralFeed.module.scss]: Could we also set our color for `p` and links inside it here? 
image

File: src/pages/feed/RecentsFeed/RecentsFeed.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          )}
+        />
+      </div>
+      <div className={styles.scrollable}>
+        <SidecarItems
+          onSlugSelect={onSlugSelect}
+          onProjectClick={onSlugSelect}
+          classes={styles}
+          params={{
+            gainersTitle: 'Gainers and loosers'
```
Comment (haritonasty) [inline src/pages/feed/RecentsFeed/RecentsFeed.js]: Let's use one heading for that layout on all pages. We used 'Social gainers and losers` in other places
@Partysun ?

File: src/pages/feed/RecentsFeed/RecentsFeed.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+          }}
+        />
+      </div>
+    </div>
+  )
+}
+
+const mapDispatchToProps = dispatch => ({
+  onSlugSelect: ({ slug }) => {
+    dispatch(push('/projects/' + slug))
```
Comment (haritonasty) [inline src/pages/feed/RecentsFeed/RecentsFeed.js]: Could you check this again, pls. Doesn't work for me, when I choose asset from popup

File: src/pages/feed/RecentsFeed/RecentsFeed.module.scss [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.searchIcon {
+  margin-left: auto;
+  fill: var(--casper);
+}
+
+.search {
+  display: flex;
+  color: $rhino;
```
Comment (haritonasty) [inline src/pages/feed/RecentsFeed/RecentsFeed.module.scss]: `var(--rhino)`

**Comment by haritonasty** [comment]
Still problems with `20 hours ago by` (should be on separate row)
and with `Recently watchlists` in night mode (also this one have another font), check pls
image

**Comment by haritonasty** [comment]
Will be more pretty, if you add:
- small margin before `img` from server, 10px will be enough, I think. 
- add `border-radius: 4px` for image

and start link from new line

image

File: src/pages/feed/RecentsFeed/RecentsFeed.js (hunk 4 above)
Comment (haritonasty) [reply src/pages/feed/RecentsFeed/RecentsFeed.js]: @GarageInc Alan approved using the same naming: `Social gainers and losers`

**Comment by GarageInc** [comment]
@haritonasty , thank you for detailed review.

**Pull request merged**